src/dparse/parser.d

```diff
@@ -1770,6 +1772,7 @@ class Parser
         if (currentIs(tok!":") || currentIs(tok!"{"))
         {
             immutable bool brace = advance() == tok!"{";
+            node.useBraceForTrueDeclarations = brace;
```

Comment:
for consistency this should be able to describe if current is `:`

Maybe add `enum StatementBlockStyle : ubyte { single, colon, block }` with some small ddoc explaining the items